airflow/api_connexion/openapi/v1.yaml

```diff
@@ -3014,6 +3171,15 @@ components:
           description: Custom notes left by users for this Dag Run.
           type: string
 
+    DeleteDatasetDagRunQueue:
+      type: object
+      properties:
+        cutoff_time:
+          description: The cutoff time of DatasetDagQueue.
+          format: date-time
+          type: string
+          nullable: true
```

Comment:
The default missing value is *now* but that’s currently not described anywhere. Or do we really need to have a default instead of making the parameter required?

Reply:
After a second thought, I don't think we really need a default value. When not provided, we just delete the DDRQ